src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/UsersTest.kt

```diff
@@ -0,0 +1,47 @@
+package fi.metatavu.timebank.api.test.functional.tests
+
+import com.github.tomakehurst.wiremock.stubbing.Scenario.STARTED
+import fi.metatavu.timebank.api.test.functional.data.TestDateUtils.Companion.getThirtyDaysAgo
+import fi.metatavu.timebank.api.test.functional.resources.LocalTestProfile
+import fi.metatavu.timebank.api.test.functional.resources.TestWiremockResource
+import io.quarkus.test.common.QuarkusTestResource
+import io.quarkus.test.junit.QuarkusTest
+import io.quarkus.test.junit.TestProfile
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Assertions.*
+
+@QuarkusTest
+@QuarkusTestResource.List(
+    QuarkusTestResource(TestWiremockResource::class)
+)
+@TestProfile(LocalTestProfile::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class UsersTest: AbstractTest() {
+
+    /**
+     * Resets Wiremock scenario states before each test
+     */
+    @BeforeEach
+    fun resetScenariosBeforeEach(){
+        resetScenarios()
```

Comment:
Wiremock isn't being used by this test class (at least for the time being). This is therefore redundant.